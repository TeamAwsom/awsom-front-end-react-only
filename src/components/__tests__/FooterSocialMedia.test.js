import React from 'react';
import FooterSocialMedia from '../FooterSocialMedia';
import renderer from 'react-test-renderer';

describe('FooterSocialMedia', () => {
  it('renders as expected', () => {
    const tree = renderer.create(<FooterSocialMedia />).toJSON();

    console.log(tree);
    expect(tree).toMatchSnapshot();
  });
});
