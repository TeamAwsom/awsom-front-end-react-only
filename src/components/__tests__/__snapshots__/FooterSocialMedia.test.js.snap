// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FooterSocialMedia renders as expected 1`] = `
<div
  className="footer_socialmedia"
>
  <div
    className="footer_socialmedia_container"
  >
    <span
      className="fa-stack fa-2x"
    >
      <a
        href="https://www.facebook.com/awsomWA"
        rel="noopener noreferrer"
        target="_blank"
      >
        <i
          className="fas fa-circle fa-stack-2x"
        />
        <i
          className="fab fa-facebook-f fa-stack-1x"
        />
      </a>
    </span>
    <span
      className="footer_socialmedia_tab"
    />
    <span
      className="fa-stack fa-2x"
    >
      <a
        href="http://www.yelp.com/biz/the-andrew-white-school-of-music-kenmore-3"
        rel="noopener noreferrer"
        target="_blank"
      >
        <i
          className="fas fa-circle fa-stack-2x"
        />
        <i
          className="fab fa-yelp fa-stack-1x"
        />
      </a>
    </span>
    <span
      className="footer_socialmedia_tab"
    />
    <span
      className="fa-stack fa-2x"
    >
      <a
        href="https://www.instagram.com/awsomusic/"
        rel="noopener noreferrer"
        target="_blank"
      >
        <i
          className="fas fa-circle fa-stack-2x"
        />
        <i
          className="fab fa-instagram fa-stack-1x"
        />
      </a>
    </span>
    <span
      className="footer_socialmedia_tab"
    />
    <span
      className="fa-stack fa-2x"
    >
      <a
        href="https://www.linkedin.com/company/andrew-white-school-of-music/?trk=biz-companies-cym"
        rel="noopener noreferrer"
        target="_blank"
      >
        <i
          className="fas fa-circle fa-stack-2x"
        />
        <i
          className="fab fa-linkedin-in fa-stack-1x"
        />
      </a>
    </span>
  </div>
</div>
`;
